.github/workflows/main.yml

```
@@ -82,3 +82,11 @@ jobs:
       - name: Mark the job as unsuccessful
         if: contains(needs.*.result, 'failure') || contains(needs.*.result, 'cancelled')
         run: exit 1
+  fmt:
+    name: Rustfmt
```

This is a nit but `rustfmt` is the name of the tool.

```suggestion
    name: Run `rustfmt`
```